test_cosi.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
-# Set to a non-empty value to print debugging messages
-DBG_RUN=1
-. $GOPATH/src/gopkg.in/dedis/cothority.v0/app/libtest.sh
+DBG_SHOW=1
+# Debug-level for app
+DBG_APP=2
+# Uncomment to build in local dir
 STATICDIR=test
 
+. $GOPATH/src/github.com/dedis/cothority/app/lib/test/libtest.sh
```

Comment:
yes, it should exist. `cosi` depends on `cothority`, so that file should be there.